src/lib/ngsi/SubscribeError.cpp

```diff
@@ -44,6 +44,45 @@ SubscribeError::SubscribeError()
 
 /* ****************************************************************************
 *
+* SubscribeError::toJson -
+*/
+std::string SubscribeError::toJson(RequestType requestType, bool comma)
+{
+  std::string  out;
+
+  out += JSON_VALUE("error", errorCode.reasonPhrase.c_str());
+  out += ",";
+  out += JSON_VALUE("description", errorCode.details.c_str());
+
+
+  if (requestType == UpdateContextSubscription)
+  {
+    //
+    // NOTE: the subscriptionId must have come from the request.
+    //       If the field is empty, we are in unit tests and I here set it to all zeroes
+    //
+    if (subscriptionId.get() == "")
+    {
+      subscriptionId.set("000000000000000000000000");
+    }
+    out += ",";
+    out += JSON_PROP("affectedItems") + "[" + JSON_STR(subscriptionId.toJson(requestType, true)) + "]";
+ }
```

Comment:
Indentation (one space seems to be missing)

Reply:
Fixed in 7480bca